### DISCUSSION TITLE
Slug based relationships

### DISCUSSION THREAD

**Discussion by benfavre** [pods-framework/pods#6643] [Ideas]
Allow relationships to be defined using the related pod's slug.

This allow's data to be moved to and from different WordPress sites, while preserving functional relationships (Synchronising data from multiple sources) .

**Comment by sc0ttkclark** [comment]
Closing this as done until I hear otherwise

**Comment by benfavre** [comment]
Been doing some testing, using an id works fine but slugs just fail, not sure how to go about debugging this either.
